Core/src/IfcGeometryConverter/FaceConverter.h

```diff
@@ -290,21 +273,21 @@ namespace OpenInfraPlatform {
 					}
 					*/
 
+					return std::shared_ptr<carve::input::PolylineSetData> polylineData = std::shared_ptr<carve::input::PolylineSetData>();

```

Comment:
This line is not necessary. Delete such lines, if the function merely throws ...

Reply:
Commented out these return lines, to be used when the function is implemented. 
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/151/commits/73af29a356a87e8ecac7b350126433b3807050e4